tests/shutdown/shutdown.pm

```diff
@@ -13,50 +13,25 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
-# Summary: support for saving and loading of hdd image
-# Maintainer: Vladimir Nadvornik <nadvornik@suse.cz>
+# Summary: Shut down the system
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
 
 use strict;
-use base 'basetest';
+use base "opensusebasetest";
 use testapi;
-use serial_terminal 'add_serial_console';
 use utils;
-use version_utils;
 
 sub run {
-    select_console('root-console');
-    if (get_var('DROP_PERSISTENT_NET_RULES')) {
-        type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules\n";
-    }
-    if (!sle_version_at_least('12-SP2') && check_var('VIRTIO_CONSOLE', 1)) {
-        add_serial_console('hvc0');
-    }
-    # Proceed with dhcp cleanup on qemu backend only
-    if (check_var('BACKEND', 'qemu')) {
-        my $network_status = script_output('systemctl status network');
-        # Do dhcp cleanup for wicked
-        if ($network_status =~ /wicked/) {
-            systemctl 'stop network.service';
-            systemctl 'stop wickedd.service';
-            assert_script_run('ls /var/lib/wicked/');
-            save_screenshot;
-            assert_script_run('rm -f /var/lib/wicked/*.xml');
-        }
-    }
     power_action('poweroff');
 }
 
-sub test_flags {
-    return {fatal => 1};
+sub post_fail_hook {
+    my ($self) = shift;
+    $self->SUPER::post_fail_hook;
 }
```

Comment:
If you don't add anything here then you also don't need to overwrite the parent post_fail_hook